kubectl-minio/cmd/tenant-expand.go

```diff
@@ -96,6 +96,11 @@ func (v *expandCmd) validate(args []string) error {
 	if args[0] == "" {
 		return errors.New("provide the name of the tenant, e.g. 'kubectl minio tenant expand tenant1'")
 	}
+	// Tenant name should have DNS token restrictions
+	if err := helpers.CheckValidTenantName(args[0]); err != nil {
```

Comment:
This might be a problem for existing tenants with an invalid name. 

For example, if the tenant `mt` was created using an older version of the plugin and they cannot expand using the newer plugin.

Reply:
There will no existing tenants in working state @kanagarajkm since DNS cannot be created for them

Reply:
However lower limit can be fixed.